docker/run_expo.sh

```diff
@@ -0,0 +1 @@
+docker run -p 19001:19001 -p 19002:19002 -p 19003:19003 -p 19004:19004 -p 19005:19005 -p 19006:19006 -v /docker_dir/project/CAMA-CORE-Example/docker:/CAMA/ -ti --name healthhacker healthhacker /bin/bash npm run web
```

Comment:
This line runs into the following error:
/usr/local/bin/npm: npm: line 2: syntax error near unexpected token ;' /usr/local/bin/npm: npm: line 2: ;(function () { // wrapper in case we're in module_context mode'

Reply:
Found a workaround + a way too open it in vscode based on https://github.com/healthhackersER/CAMA-CORE-Example/issues/12

@wooramkang either you add the files below to your PR
or I do a PR as soon as this PR is merged

**docker-compose.yml**
version: '3.7'
 
services:
  testexpo:
    image: healthhacker:latest
    command: "npm run web"
    expose:
      - "19000"
      - "19001"
      - "19002"
    ports:
      - "19000:19000"
      - "19001:19001"
      - "19002:19002"
    volumes:
      - myexpo:/home/node/src
volumes:
  myexpo:

##volumes:
#volumes: Containers are short lived and stopping the container will loose you data.
#For e.g. once the container is up we might install npm packages. 
#If the packages are not able to persist then we need to reinstall packages every time we start the container. 
#In order to persist the packages and changes, docker-compose creates a named volume and 
#keep the files of /home/dev/src in the volume and can be accessible even after docker restart.

#!Keep in mind ‘docker-compose down’ will remove the volume and you need to reinstall all the packages again.

**.devcontainer/devcontainer.json**
{
    "name": "expo-test",
    "dockerComposeFile": "../docker-compose.yml",
    "service": "testexpo",
    "workspaceFolder": "/CAMA/CAMA-CORE-Example/",
    "extensions": [
       "esbenp.prettier-vscode"
    ],
    "forwardPorts": [ 19000, 19001, 19002, 19006, 19007 ],
    "shutdownAction": "stopCompose"
}

**To run it:**
Open VS Code in source folder. VS Code will prompt to Reopen in Container, click Reopen in Container button. Wait for some time, and VS Code will launch from the container.

Open a new shell inside vscode
execute: npm run web

navigate to localhost:19007

**Bonus**:
You can now develop in the container.
Once launched in container, all your code changes will be available in the container. 
Make sure to push your changes to git before exiting the container.

Reply:
Might be an overkill for that PR but if you decide to go with my docker compose.
You could modify it to include this repo 
https://github.com/pinfake/android-emulator-x11 (see https://github.com/healthhackersER/CAMA-CORE-Example/issues/34) 

so you get a virtual device when launching the docker container.

Reply:
if you open it with ip:19007 or something, not localhost:19007, then it would work

Reply:
i don't know why this happen so far. it need to be figured out

Reply:
and i'll remove unnecessary port numbers